### ISSUE TITLE
[WebUI] Wrong CPU temperature on Elkhart Lake devices

### ISSUE THREAD

**Issue by GvY85** [pi-hole/pi-hole#5161]
### Versions
- Pi-hole:  5.15.3
- AdminLTE:  5.18.3
- FTL:  5.20.1

### Platform
- OS and version:  Debian 11 with DietPi 
- Platform:  x86_64 Intel Elkhart Lake, J6426 CPU

### Expected behaviour

Correct CPU temp in web interface

### Actual behaviour / bug

Currently the `Temp:` field shows `26.8*C` while running the DietPi command `cpu` shows `33*C`.
It seems reasonable to assume the highest of these is the actual CPU die temperature and the web interface is showing a different temperature sensors value.

## Debug Token

- URL:  https://tricorder.pi-hole.net/isfMrOyE/

### Screenshots

![image](https://user-images.githubusercontent.com/38404393/216763950-7f0fa56c-9f3a-477b-9877-59a9b1425b3e.png)

![image](https://user-images.githubusercontent.com/38404393/216763966-24324121-d88e-4b0d-8c78-4263452d9a4e.png)

Thank you.

**Comment by DL6ER** [comment]
> It seems reasonable to assume the highest of these is the actual CPU die temperature 

No. We have already seen several devices with unconnected chassis sensors reporting almost random temperatures, often in the 80-100°C area where you could also expect a very hot CPU.

My vote is still removing the temperature display and make a high load value (high CPU %) more prominent. This is the cause of issues, the temperature is almost always just an indirect measure of this.

**Comment by jfb-pihole** [comment]
I am in favor of removing the CPU temperature.  I am ambivalent on the load display.  In many cases, the underlying host has recently completed a CPU intensive task such as rebuilding gravity, and it is not abnormal to see higher CPU load during that time.  But, when the dashboard warns the user of this, we frequently get help requests.

Rarely have we seen a performance issue with small and temporary CPU load spikes.

My preference would be not to make the CPU load value more prominent, unless it is clearly excessive and unusually high.

**Comment by DL6ER** [comment]
The load values are 1, 5, and 15 minute averages, respectively. One can say that the system is not able to handle things when the load is larger than the number of cores (rule of thumb). I'd say it's justified to assume Pi-hole operation is degraded when the 15 minutes average exceeds this level as it ultimately means there is too much to be done on this device and every process (including `pihole-FTL`) will be queued for CPU time.
